### ISSUE TITLE
Why using a AsyncJob instead of a Future directly?

### ISSUE THREAD

**Issue by alanjds** [dgkim5360/asyncloop#2]
https://github.com/dgkim5360/asyncloop/blob/228a7e9644e6f3e0705494a6d5c4ad4c857bd8ea/asyncloop/job.py#L21

Looks like the AsyncJob does not add much to the Future implementation. Instead, it proxies almost all of its calls.

Could be changed the implementation to use Future directly or at least subclass it? What you think?

**Comment by dgkim5360** [comment]
I agree with your point. It seems that I had no brake for enough consideration, a half year ago.

In my current opinion, there's no need to implement a custom Future-like class here. And it would be the best to use a native `concurrent.Future`directly.

**Comment by dgkim5360** [comment]
I add a commit which changes the following.
1. `AsyncLoop.submit` returns `Future` object directly.
2. `AsyncJob` is replaced with the simpler object `PendingJob` to be stored in the pending queue `AsyncLoop.pending`.

What do you think? Any feedback would be appreciated.
(If you want to review before the commit, I will do PR next time.)
